src/vmemcache_repl.c

```diff
@@ -232,7 +269,7 @@ repl_p_lru_insert(struct repl_p_head *head, void *element,
 
 	ASSERTne(ptr_entry, NULL);
 	entry->ptr_entry = ptr_entry;
-	*(entry->ptr_entry) = entry;
+	util_bool_compare_and_swap64(entry->ptr_entry, NULL, entry);
```

Comment:
ditto

Reply:
It has to be true. I can add assert() here.